airbyte-scripts/github-filler/main.py

```diff
@@ -0,0 +1,141 @@
+import json
+import logging
+import sys
+from typing import List, Optional
+
+import requests
+
+logging.basicConfig(level=logging.INFO)
+
+
+def logger(func):
+    def wrapper(*args):
+        logging.info(f'Function: {func.__name__}')
+        response = func(*args)
+        logging.info(f'Response status: {response.status_code}, response body: {"Success" if response.status_code in [200, 201] else response.content}')
+    return wrapper
+
+
+def get_headers(token: str):
+    return {
+        'Authorization': f'Bearer {token}',
+        'Accept': 'application/vnd.github.v3+json application/vnd.github.inertia-preview+json',
+    }
+
+
+def get_all_branches(rep, head):
+    url = f"https://api.github.com/repos/{rep}/branches"
+    response = requests.get(url=url, headers=head)
+    return response.json()
+
+
+def pull_requests(rep: str, branches: list, head: dict):
```

Comment:
`def create_pull_requests(rep: str, branches: list, headers: dict):`

btw headers will not be required if you use class and self.session